diff --git a/cmake/cef_variables.cmake b/cmake/cef_variables.cmake
index 03f7334..5e9add7 100644
--- a/cmake/cef_variables.cmake
+++ b/cmake/cef_variables.cmake
@@ -392,7 +392,7 @@ if(OS_WINDOWS)
   set(CEF_DEBUG_INFO_FLAG "/Zi" CACHE STRING "Optional flag specifying specific /Z flag to use")
 
   # Consumers using different runtime types may want to pass different flags
-  set(CEF_RUNTIME_LIBRARY_FLAG "/MT" CACHE STRING "Optional flag specifying which runtime to use")
+  set(CEF_RUNTIME_LIBRARY_FLAG "/MD" CACHE STRING "Optional flag specifying which runtime to use")
   if (CEF_RUNTIME_LIBRARY_FLAG)
     list(APPEND CEF_COMPILER_FLAGS_DEBUG ${CEF_RUNTIME_LIBRARY_FLAG}d)
     list(APPEND CEF_COMPILER_FLAGS_RELEASE ${CEF_RUNTIME_LIBRARY_FLAG})
